app/views/pages/terms_and_conditions.html.erb

```diff
@@ -0,0 +1,218 @@
+                </ul>
+            <p>
+                As a school, it's your responsibility to decide if a school experience candidate complies with your DBS and safeguarding policies.
+            </p>
+        <h3 class="govuk-heading-s">Access to the Service</h3>
+            <p>
+                We will not be liable if for any reason the Service is unavailable at any time or for any period. From time to time, we may restrict access to all or some parts of the Service to users who have registered with us.
+            </p>
+            <p>
+                If you choose, or are provided with, a user identification code, password or any other piece of information as part of our security procedures, you must treat such information as confidential and you must not disclose it to any third party.
+            </p>
+            <p>
+                We reserve the right to restrict or deny you access to all or some parts of the Service if, in our opinion, you've failed to comply with these General Terms.
+            </p>
+            <p>
+                We retain the right to withdraw, without notice, school experience listings that breach these terms and to withdraw the access of users who breach them.
+            </p>
+        <h3 class="govuk-heading-s">Listing data</h3>
+            <p>
+                Data ownership: schools have sole responsibility for maintaining the accuracy and appropriateness of the data contained in their listings.
+            </p>
+        <h3 class="govuk-heading-s">Department for Education (DfE) responsibility</h3>
+            <p>
+                The DfE does not make any decisions on individual applications.
+            </p>
+            <p>
+                The DfE does not investigate complaints about individual applications.
+            </p>
+        <h3 class="govuk-heading-s">Other policies</h3>
+            <p>
+                You can <%= link_to 'make a complaint', 'https://form.education.gov.uk/service/Contact_the_Department_for_Education' %> or <%= link_to 'give feedback about this service', 'https://schoolexperience.education.gov.uk/candidates/feedbacks/new' %>.
+            </p>
+            <p>
+                Our <%= link_to 'privacy notice', 'https://schoolexperience.education.gov.uk/privacy_policy' %> explains how we keep your data safe.
+            </p>
+        <h3 class="govuk-heading-s">Last updated</h3>
+            <p>
+                We may need to update these General Terms so we recommend that you revisit this information from time to time.
+            </p>
+            <p>
+                If you continue to use our service after the terms and conditions are updated, you agree to any of the changes we’ve made.
+            </p>
+            <p>
+                This version was last updated on 23 April 2024.
+            </p>
+    <br>
```

Comment:
The html is not well-formed - need to close the wrapping `<div>` and `<section>` tags.

Don't use line breaks `<br>` unless absolutely required.